woocommerce/wc-admin
# Add Homescreen layout control

Fixes #5224.

This PR seeks to implement the Homescreen layout control described in the attached issue.

The following changes were made:
- Introduction of the `woocommerce_admin_homepage_layout` user meta field
- Replacement of the `hasInbox` class with `two-columns` (controlled by `woocommerce_admin_homepage_layout`)
- Addition of "Display" item to activity panel (note - it's not implemented as a `Tab` since those are being removed)
- Tweaking of styling to match Figma - namely making the single column wider

### Accessibility




### Screenshots

<img width="293" alt="Screen Shot 2020-10-20 at 1 10 09 PM" src="https://user-images.githubusercontent.com/63922/96622860-7e87d880-12d8-11eb-88cc-6fd2386cacaf.png">

### Detailed test instructions:

- Verify that the "display" button/tab is only shown on the Homescreen
- Verify that "two columns" is the default
- Verify that selecting "single column" changes the layout, and persists after a refresh



### Changelog Note:



Enhancement: add ability to toggle homescreen layouts.
